src/popup/pages/SignTransaction.vue

```diff
@@ -51,8 +58,24 @@ export default defineComponent({
           return;
         }
 
+        const replaceCallerTx = async (tx: Encoded.Transaction) => {
+          const unpackedTx = unpackTx(tx);
+          if (unpackedTx.tag === Tag.ContractCallTx || unpackedTx.tag === Tag.ContractCreateTx) {
+            // TxUnpacked isn't exported
+            // typescript is unaware that ContractCallTx and ContractCreateTx do have a callerId
+            (unpackedTx as any & { callerId: Encoded.AccountAddress })
+              .callerId = activeAccount?.address as Encoded.AccountAddress;
+            unpackedTx.nonce = (await aeSdk.api.getAccountByPubkey(
+              activeAccount?.address as Encoded.AccountAddress,
+            )).nonce + 1;
```

Comment:
Denis was suggesting to use `nextNonce` instead.
https://github.com/superhero-com/superhero-wallet/pull/2782/files#diff-9e41b71bd3f413859c81a2cd9ebfe4633d90f9241c6f99c1af35709cf5594a55R168